frontend/appflowy_flutter/lib/util/color_analyzer/color_analyzer.dart

```diff
@@ -0,0 +1,50 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+
+class ColorAnalyzer {
```

Comment:
Final thing, can you add a unit test to this location:
`frontend/appflowy_flutter/test/unit_test/color_analyzer_test.dart`

Where you test it with some predefined colors.

Let me know